frontend/src/features/contacts/contact/create/Person/CreatePersonForm.tsx

```diff
@@ -162,6 +163,17 @@ const CreatePersonComponent: React.FC<FormikProps<ICreatePersonForm>> = ({
     }
   };
 
+  const isContactMethodInvalid = useMemo(() => {
```

Comment:
Otherwise the custom validation will change the text to red before user attempts to save etc.